client/beefy/src/worker.rs

```diff
@@ -638,6 +645,7 @@ where
 				.expect("forwards closure result; the closure always returns Ok; qed.");
 		} else {
 			debug!(target: LOG_TARGET, "🥩 Can't set best beefy to older: {}", block_num);
+			metric_set!(self, beefy_best_block_to_old_block, block_num);
```

Comment:
nit pertaining preexisting log message: block's not strictly older, just not newer. `"🥩 Can't set best beefy to old: {}"` would already be more accurate.